test/api/bucket/api_test.go

```diff
@@ -0,0 +1,135 @@
+package bucket
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/test/assert"
+)
+
+func TestBucketCRUD(t *testing.T) {
+	for _, bucket := range Buckets {
+
+		t.Run("Create Bucket and Compare", func(t *testing.T) {
```

Comment:
Few notes on this test:
- I'm not sure how much it is needed, but likely it is needed to test various sample files Create/Get/Index, but in case of Bucket.Get it should check not only Path, but also content or content length.
- Nit, but when iterating over samples in subtest, add their name/Path in this case (same for line 46).